### DISCUSSION TITLE
Constant Merge Conflicts with CollabNet Desktop but not with Default (Subclipse)

### DISCUSSION THREAD

**Discussion by meldorq** [subclipse/subclipse#187] [Q&A]
Hi, I have one single project which refuses to be merged from on branch to another when I use CollabNet Desktop as merge 
implementation in Subclipse 4.3.3 with JavaHL 1.13 in Ubuntu 20.04.

The merge runs just fine with the default Subclipse merge implementation. But the CollabNet Desktop is much more convenient when it comes to selecting revisions, so I'd rather use that.

All I get as an error message is
"Unresolved conflicts from previous merge detected
One or more conflicts were produced during a previous merge of [my project]"

But I am sure that there are no previous merge conflicts. This error occurs even after a fresh checkout of the project.

I don't see anything related in the eclipse error log.

I guess it's a bug in CollabNet Desktop (external lib?), but I have no idea where I can report it.

Any ideas how to solve it or how to obtain more information?

**Comment by markphip** [comment]
The Merge Client is just an alternate UI that was developed. Both UI end up calling the same svn merge API and all of the actual work is done by Subversion itself.

I would check the console, but I would guess that what is happening is that SVN is breaking the merge up into multiple merges because of the revision ranges you selected and/or the merge tracking information in your repository. The console will show this. What is happening is the merge of let's say the first revision range ends with some conflicts so it cannot continue to merge the next range.

**Reply by meldorq** [reply]
The CollabNet client doesn't write anything to the SVN console when this happens. There is only the modal with the error message, nothing else.

Edit: This indicates IMHO that the CollabNet client does something before contacting SVN. Maybe it stumbles over some stale cached information? Deleting the project and checking if fresh out of SVN doesn't help, so it's likely not that.

**Reply by NorbertSandor** [reply]
I have run into the same issue.
No further info on the console.
The error happens even trying to merge after a clean checkout.
Is there a solution or any hint for further investigation?

I was able to merge successfully without conflicts by using the `svn merge` command directly.
After committing the merged changes, merging started to work in Eclipse, too.
I am sure that this was an Eclipse CollabNet client problem...

**Comment by t14z** [comment]
你好！你的邮件已收到
